docker/seccomp.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+def loadDefault( filename ):
+    with open( filename, 'r' ) as f:
+        return json.load( f )
+
+def inPlaceTransform( jdata ):
+    # for names_action_etc in jdata[ 'syscalls' ]:
+    #     names = names_action_etc[ 'names' ]
+    #     action = names_action_etc[ 'action' ]
+    #     if 'personality' in names:
+    #         print( action )
+    jdata[ 'syscalls' ].extend(
+        { 
+            "names": [ "personality" ],
+            "action": "SCMP_ACT_ALLOW",
+            "args": [ { "index": 0, "value": value, "op": "SCMP_CMP_EQ" } ]
+        }
+        for value in [ 0x0040000, 0x0400000, 0x0440000 ]
```

Comment:
Can these be extracted out to meaningfully named constants? Perhaps using an `Enum`?

Reply:
Done